src/java.base/share/classes/java/util/zip/ZipOutputStream.java

```diff
@@ -258,54 +258,53 @@ public void closeEntry() throws IOException {
         if (current != null) {
             ZipEntry e = current.entry;
             switch (e.method) {
-            case DEFLATED:
-                def.finish();
-                while (!def.finished()) {
-                    deflate();
+                case DEFLATED -> {
```

Comment:
Same remark here - it's not clear what's going on.

Reply:
Here, it's mostly indentation change (+4 spaces). You can set "Hide whitespace changes" and see:
![image](https://user-images.githubusercontent.com/5114450/119436860-3259b680-bd47-11eb-92d8-f940493d08c2.png)
Alternatively, I can indent back this switch, placing `case` on the same indentation level as `switch`. I'm not sure about recommended code style in OpenJDK project, it looks like it's not very consistent.